regression-test/data/dynamic_table_p0/sql/q02.out

```diff
@@ -18,15 +18,15 @@
 4060
 
 -- !q02_7 --
-['.805', '.621']
```

Comment:
the behavior seems different with previous, what if user use the previous behavior? will this lead to incompatible？

Reply:
this is only in array string , for standard in json , ['aa', 'bb'] is wrong expr, and I met a user issue for changing the array output, I think we should make array string is right!